src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -351,19 +351,28 @@ describe('lib/viewers/image/ImageViewer', () => {
             expect(image.annotationControls).toBeInstanceOf(AnnotationControls);
         });
 
-        test('should call annotations controls init with callbacks', () => {
-            jest.spyOn(image, 'areNewAnnotationsEnabled').mockReturnValue(true);
-            jest.spyOn(image, 'hasAnnotationCreatePermission').mockReturnValue(true);
-            jest.spyOn(AnnotationControls.prototype, 'init').mockImplementation();
-
-            image.loadUI();
-
-            expect(AnnotationControls.prototype.init).toBeCalledWith({
-                fileId: image.options.file.id,
-                onClick: image.handleAnnotationControlsClick,
-                onEscape: image.handleAnnotationControlsEscape,
-            });
-        });
+        test.each`
+            enableAnnotationsImageDiscoverability | initialMode              | should
+            ${false}                              | ${AnnotationMode.NONE}   | ${'AnnotationMode.NONE if enableAnnotationsImageDiscoverability is false'}
+            ${true}                               | ${AnnotationMode.REGION} | ${'AnnotationMode.REGION if enableAnnotationsImageDiscoverability is true '}
+        `(
+            'should call annotation controls init with callbacks and with initialMode set to $should ',
```

Comment:
is the `should` param needed? Could it just be `should call annotation controls init with $initialMode when enableAnnotationsImageDiscoverability is $enableAnnotationsImageDiscoverability`?

Reply:
Not needed, will remove.